Clients/GUI/Authentication/AuthStateProvider.cs

```diff
@@ -15,9 +15,8 @@ public class AuthStateProvider : AuthenticationStateProvider
         /// </summary>
         public async override Task<AuthenticationState> GetAuthenticationStateAsync()
         {
-            var anonymous = new ClaimsPrincipal(new ClaimsIdentity());
-
-            return await Task.FromResult(new AuthenticationState(anonymous));
+            var notAnonymous = new ClaimsPrincipal(new ClaimsIdentity("test"));
```

Comment:
Can you  please explain what is "claims"?

Reply:
https://docs.microsoft.com/en-us/aspnet/core/security/blazor/?view=aspnetcore-3.1&tabs=visual-studio

> Can you please explain what is "claims"?